packages/manager/modules/pci/src/components/project/images-list/images-list.service.js

```diff
@@ -3,12 +3,71 @@ import groupBy from 'lodash/groupBy';
 import map from 'lodash/map';
 import pick from 'lodash/pick';
 import omit from 'lodash/omit';
+import snakeCase from 'lodash/snakeCase';
 
 import Image from './images.class';
 
+function getDistribution(name, type) {
+  const regex = {
```

Comment:
Please move this regex definition to a constant file. Now its getting initialized every time getDistribution method is called.

Reply:
Fixed